packages/components/src/confirm-dialog/test/index.tsx

```diff
@@ -349,5 +349,73 @@ describe( 'Confirm', () => {
 
 			expect( onConfirm ).toHaveBeenCalled();
 		} );
+
+		it( 'should set confirm button busy and disable both buttons when `isBusy` is true', () => {
```

Comment:
I'm not sure how much value these tests add, but will leave this for @mirka or @aduth.

Reply:
Thanks! If there's agreement that these tests don’t add much value, I’m happy to remove them.